extensions/collected-notes/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+	"printWidth": 120,
+	"singleQuote": false,
+	"useTabs": true
```

Comment:
```suggestion
```
Could you remove this so we have the same settings for all extensions, it might require a fix-lint to be ran from package.json 🙂